docs/data/joy/components/alert/alert.md

```diff
@@ -64,12 +102,11 @@ Use the `startDecorator` and/or `endDecorator` props to insert actionable button
 
 ## Accessibility
 
-Here are a few tips to make sure you have an accessible alert component:
-
-- When the component is dynamically displayed, the content is automatically announced by most screen readers.
-  At this time, screen readers do not inform users of alerts that are present when the page loads.
-
-- Using color to add meaning only provides a visual indication, which will not be conveyed to users of assistive technologies such as screen readers.
-  Ensure that information denoted by the color is either obvious from the content itself (for example the visible text), or is included through alternative means, such as additional hidden text.
+Here are some factors to consider to ensure that your Alert is accessible:
 
-- Actions must have a tab index of 0 so that they can be reached by keyboard-only users.
+- Because alerts are not intended to interfere with the use of the app, your Alert component should _never_ affect the keyboard focus.
+- If an alert contains an action, that action must have a `tabindex` of `0` so it can be reached by keyboard-only users.
+- Alerts should not disappear automatically—[timed interactions](https://www.w3.org/TR/UNDERSTANDING-WCAG20/time-limits-no-exceptions.html) can make your app inaccessible to users who need extra time to understand or locate the alert.
```

Comment:
I think that this misses the nuance of "essential"

> [2.2.3](http://www.w3.org/TR/2008/REC-WCAG20-20081211/#time-limits-no-exceptions) No Timing: Timing is not an [essential](https://www.w3.org/TR/UNDERSTANDING-WCAG20/time-limits-no-exceptions.html#essentialdef) part of the event or activity presented by the content, except for non-interactive [synchronized media](https://www.w3.org/TR/UNDERSTANDING-WCAG20/time-limits-no-exceptions.html#synchronizedmediadef) and [real-time events](https://www.w3.org/TR/UNDERSTANDING-WCAG20/time-limits-no-exceptions.html#real-time-eventsdef). (Level AAA)

https://www.w3.org/TR/UNDERSTANDING-WCAG20/time-limits-no-exceptions.html

I think that we should nuance it. It's also a AAA level which isn't meant for all websites (we focus more on AA)

Reply:
That makes sense. If our goal is AA then this may not be applicable (I don't see anything that mentions timing for AA standards) and we could remove it, or else note that this would apply to AAA.